tests/qemu/linux/test_dnsmasq_server.cpp

```diff
@@ -46,9 +46,10 @@ namespace
 {
 struct CapturingLogger : public mp::logging::Logger
 {
-    void log(mpl::Level level, mpl::CString category, mpl::CString message) const override
+    void log(mpl::Level level, std::string_view category, std::string_view message) const override
     {
-        logged_lines.push_back(message.c_str());
+        (void)category;
+        logged_lines.emplace_back(message);
```

Comment:
I think omitting the name from the parameter list would avoid the unused warning, but this also works so whichever you prefer

Reply:
I like keeping the variable name, but it can also be done by commenting out the variable name in-line. That would look cleaner.

Reply:
Done